lookout/style/typos/symspell.py

```diff
@@ -123,21 +123,14 @@ def load_dictionary(self, corpus):
         File should contain space-separated word-count pairs one at a line.
 
         Args:
-            corpus (str): Path to corpus file.
+            corpus (str): Path to .csv corpus file.
         """
         if os.path.exists(corpus):
-            try:
-                with lzma.open(corpus, "rt") as f:
-                    for line in f:
-                        key, count = line.split()
-                        count = int(count)
-                        self._create_dictionary_entry(key, count)
-            except lzma.LZMAError:
-                with open(corpus, "r") as f:
-                    for line in f:
-                        key, count = line.split()
-                        count = int(count)
-                        self._create_dictionary_entry(key, count)
+            with smart_open(corpus, "r") as f:
+                for line in f:
+                    key, count = line.split()
```

Comment:
Please use `csv.reader` here to have a completely standard format

Reply:
Maybe it's better to do in in #629 where everything is moved to `csv` library? Otherwise I will have to do same changes in different PRs.

Reply:
Ah right, because you already have the tests. ok